view/script/patient.js

```diff
@@ -47,5 +50,53 @@
         events: window.events
     });
 
+    perWeekDatesConfig = datesByWeek(allDatesConfig);
+    config.data = perWeekDatesConfig;
+
+    $('.fc-today-button').click(function todayButtonClick () {
+        perWeekDatesConfig = datesByWeek(allDatesConfig);
+        config.data = perWeekDatesConfig;
+        new Chart(ctx, config);
```

Comment:
rather than creating a `new Chart()` everytime the calendar dates change,
update the data being passed into the chart and call `Chart.update()`.

see this example for how that works
https://github.com/nnnick/Chart.js/blob/v2.0-dev/samples/line-legend.html